core/src/test/resources/internal-port-check-dockerfile/Dockerfile-bash

```diff
@@ -0,0 +1,12 @@
+# If you ever upgrade this image, watch out for bash being installed by default in newer alpine base images
```

Comment:
the warning is not relevant here, isn't it?

Reply:
True, I can remove it there